app/src/main/kotlin/com/simplemobiletools/dialer/adapters/ContactsAdapter.kt

```diff
@@ -271,7 +271,7 @@ class ContactsAdapter(
     private fun setupView(view: View, contact: Contact, holder: ViewHolder) {
         view.apply {
             setupViewBackground(activity)
-            findViewById<ConstraintLayout>(R.id.item_contact_frame).isSelected = selectedKeys.contains(contact.rawId)
+            findViewById<FrameLayout>(R.id.item_contact_frame).isSelected = selectedKeys.contains(contact.rawId)
```

Comment:
revert this back, it is a constraintlayout

Reply:
<img width="630" alt="Screenshot 2023-07-02 at 22 15 17" src="https://github.com/SimpleMobileTools/Simple-Dialer/assets/56008710/c6887550-1c64-4bed-9c51-a02153b35ad4">

How is that possible?

Reply:
My app crashes otherwise, because it's frameLayout in my file

Reply:
try updating to the latest commons